### ISSUE TITLE
Github Berechtigungen ändern

### ISSUE THREAD

**Issue by polenter** [dr-schenk/investortax24#2]
@dr-schenk 
Ich habe es in meinem Repository angeschaut

[hier...](https://github.com/dr-schenk/investortax24(settings)

...gibt es diese Einstellung ganz unten

![grafik](https://github.com/dr-schenk/investortax24/assets/29297999/bd0f89d3-0c98-4fd4-bc8e-7e3cc0e7b0a3)

**Comment by dr-schenk** [comment]
public ist ja jetzt eingestellt, aber ich kann deine rolle nicht ändern
Mit freundlichen Grüßen

Diplom-Kaufmann Univ.
Dr. Rainer Schenk
Steuerberater
Certified Tax Counsel


KANZLEI DR. SCHENK
ISO zertifizierte Steuerberater
TAX COUNSEL

Kanzleisitz:
Kleiner Burgweg 9  I  D-97645 Ostheim

Ausgelagertes Office:
Tanzberg 1 (Haus zum freien Ritter)  I  D-98617 Walldorf
Ludwigstraße 19  I D-97688 Bad Kissingen

m: +49 170 5492959
t:   +49 9777 6190 775
t:   +49 9777 8973990 10
f:   +49 32127243651
***@***.***
www.kanzleidrschenk.de

Hinweis: Etwaige Schreibfehler bitten wir zu entschuldigen. Diese sind dem Umstand geschuldet, dass wir unsere Texte mit einer automatisierten Spracherkennungssoftware erstellen.

Am 23.05.2024 um 13:23 schrieb Pawel Idzikowski ***@***.***>:



@dr-schenk<https://github.com/dr-schenk>
Ich habe es in meinem Repository angeschaut

[hier...](https://github.com/dr-schenk/investortax24(settings)

...gibt es diese Einstellung ganz unten

grafik.png (view on web)<https://github.com/dr-schenk/investortax24/assets/29297999/bd0f89d3-0c98-4fd4-bc8e-7e3cc0e7b0a3>

**Comment by polenter** [comment]
würdest du bitte noch den folgenden Benutzer einladen?

wincognito

Es ist mein weiteres Account - PRO. Vielleicht muss der eingeladene PRO sein???

**Comment by dr-schenk** [comment]
Benutzer ist eingeladen

**Comment by dr-schenk** [comment]
<img width="1247" alt="2024-05-23_15-11-55" src="https://github.com/dr-schenk/investortax24/assets/170618132/eeecf9af-ca57-4c1c-862f-953e45534850">

**Comment by dr-schenk** [comment]
immer noch nicht

**Comment by dr-schenk** [comment]
Habe auch im Netz nach Lösung vergeblich gesucht wegen Roles

**Comment by dr-schenk** [comment]
https://youtu.be/UoWKo6Kcnis?si=sRiH0OtX59tS_UfI

**Comment by dr-schenk** [comment]
ich glaube, dass du keine Einschränkungen hast >> siehe Youtube

**Comment by wincognito** [comment]
nein, nach wie vor sehe ich keine Settings. gerade schaue ich mir das Video an :-)

**Comment by wincognito** [comment]
Ich glaube, nur in github Enterprise bzw. Teams ist es möglich mehrere Admins/Owners zu haben

[Best practices for organizations](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/best-practices-for-organizations)

Die Enterprise bringt aber viel Komplexität, die unsere Produktivität u.U. bremsen kann, bzw. brauchen wir einen Voll-Pro Admin für das GitHub Management

**Comment by wincognito** [comment]
Sie melden sich selbst bei dir, nach dem Ablauf von Trial.
Irgendwann brauchst du bestimmt eine robuste Lösung, die funktioniert unabhängig davon, ob Menschen kommen oder gehen. In der früheren Kick-Off Phase wäre es IMHO überdimensioniert.

Ich würde vorschlagen, dass wir aktuell die bisherige Bahn fahren. Server ist da, Domain auch, Quelltexte haben auch ihr Heim. Wenn der Schuh druckt, können wir gegen steuern.

**Comment by dr-schenk** [comment]
Ich habe dennoch den support kontaktiert und ein Ticket eröffnen lassen, um das zu klären.

**Comment by wincognito** [comment]
Würdest du bitte die Repository zurück privat machen?
